dramatiq/middleware/middleware.py

```diff
@@ -33,39 +41,39 @@ class Middleware:
     """
 
     @property
-    def actor_options(self):
+    def actor_options(self) -> set[str]:
         """The set of options that may be configured on each actor."""
         return set()
 
     @property
-    def forks(self):
+    def forks(self) -> list[Callable[[], None]]:
```

Comment:
I’ve not dug deeper through the code, but to match Prometheus [here](https://github.com/Bogdanp/dramatiq/blob/057e03b1727c2813c6e8f5ba209bd3cbec786b0d/dramatiq/middleware/prometheus.py#L194-L206) the return type should be `int`.

Reply:
I think ultimately the fork function is called here

https://github.com/Bogdanp/dramatiq/blob/fb07c5e033f710f5f3e510aaa42ed9bd93737d77/dramatiq/cli.py#L521

So yes I agree the return type should be `int`